src/Common/src/System/PasteArguments.cs

```diff
@@ -17,8 +17,9 @@ public static string Paste(IEnumerable<string> arguments, bool pasteFirstArgumen
         {
             var stringBuilder = new StringBuilder();
 
-            foreach (string argument in arguments)
```

Comment:
I don't think "Paste" makes sense in this context, for either the class or the method name.